### ISSUE TITLE
nClam.ClamClient.SendAndScanFileAsync() always returns Unknown from remote servers

### ISSUE THREAD

**Issue by NVWhiteHat** [tekmaven/nClam#52]
When I invoke nClam.ClamClient.SendAndScanFileAsync() pointed to a ClamD service installed locally, I get ClamScanResults.Clean & ClamScanResults.VirusDetected responses every time.  But when I call a ClamD service on a remote server, exact same install, virus DBs, configs, exact same client code, I only ever get back result ClamScanResults.Unknown.  I'm developing against .Net Framework 4.6.1.  Any ideas??

**Comment by tekmaven** [comment]
There should be no difference.  Can you check what value is in the `RawResult` property when the result is Unknown?

**Comment by NVWhiteHat** [comment]
Right, there should be no difference but there is for me for some reason.  RawResult is just coming back null/nothing.  Nothing in the Clamd.log file on that remote server to indicate any problems.  Know of anywhere else I could extract some useful diagnostics to indicate what the problem is?  Much appreciated any help you might have.  I've been stuck on this for weeks now.

**Comment by NVWhiteHat** [comment]
For anyone having a similar issue, I found that if I restored the clamd.conf to the default settings (copied back from conf_examples/clamd.conf.sample, the issue went away and the ClamD service began returning Clean/VirusDetected responses.  I have no idea specifically what configuration was hosing the service, but there was something in it that caused it to only ever return Result = Unknown.

**Comment by edwardpmcgovern** [comment]
I get this same error.  Just returns Unknown (null) using SendAndScanFileAsync().  I thought it was case or the directories from what you were saying here, case of the db directory (meaning C:/ClamAV vs C:/clamav).  I don't have a conf_examples/clamd.conf.sample folder.  But I had the readme.md that had what clamd.conf should look like and copied those values into the setup file.  Same issue.  Keeps returning Unknown.
